ledger/acctupdates.go

```diff
@@ -1048,8 +1054,20 @@ func (au *accountUpdates) lookupResource(rnd basics.Round, addr basics.Address,
 			if persistedData.addrid != 0 {
 				// if we read actual data return it
 				au.baseResources.writePending(persistedData, addr)
+
+				// make the account visible on the cache immedietly
+				au.accountsMu.Lock()
+				au.baseResources.flushPendingWrites()
```

Comment:
why not adding a new `write` method that is `writePending` + `flushPendingWrites` or a some new implementation?

Reply:
It's possible to use the existing `write()` in the LRU. It is what the `flushPendingWrites()` uses internally.